src/actions/bookmark.ts

```diff
@@ -0,0 +1,21 @@
+import {FacetedCompositeUnitSpec} from 'vega-lite/build/src/spec';
+import {PlotObject} from '../models/plot';
+import {ReduxAction} from './redux-action';
+
+export type BookmarkAction = BookmarkAddPlot | BookmarkRemovePlot | BookmarkModifyNote;
+
+export const BOOKMARK_ADD_PLOT = 'BOOKMARK_ADD_PLOT';
+export type BookmarkAddPlot = ReduxAction<typeof BOOKMARK_ADD_PLOT, {
+  plot: PlotObject
```

Comment:
The reason is that plotObject is a bad name for spec + its metadata. 

Plot sounds a bit misleading as it may mean more for the spec but not fieldInfo metadata.

I think going forward we might have a better name for `PlotObject`.  
Naming it consistently will help us rename it easily.  

(Any better idea for naming `PlotObject`?)